test-scripts/tomcat-deployment/ansible/roles/tomcat/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+---

```

Comment:
Should this file have the `.yaml` extension [1] ?

[1] http://www.yaml.org/faq.html

Reply:
Fixed with 0e68a5bf790f62747c93c792686566a1312d6575